### PR TITLE
Add new example - sandbox-aws with AWS full spec

### PR DESCRIPTION
**Pull request by cmperro**

### CODE REVIEW

**Review by cmperro — COMMENTED** [review]
should be infra_region

File: examples/sandbox-aws/definition.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+###############################################################################
+#Global variables - these can also be set in your profile
+#These will take precedence over those set in your profile
+###############################################################################
+name_prefix: chpe #name prefix for everything, infra, plat, and run
+region: us-east-1 #CSP region for infra
```
Comment (Chaffelson) [inline examples/sandbox-aws/definition.yml:7]: infra_region

File: examples/sandbox-aws/definition.yml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+              #ephemeral = ephemeral disk - not EBS
+              #standard = magnetic
+              #st1 = througput optimized HDD
+              #gp2 = general purpose ssd
+              volumeType: st1
+################################################################################
+#Operational Database Configurations
+################################################################################
+opdb:
+  name: chpe-od-2 #Override OpDB name - default is <name_prefix>-od
```
Comment (Chaffelson) [inline examples/sandbox-aws/definition.yml:96]: opdb:
  definitions:
    - name: foo
    - name: bar

File: examples/sandbox-aws/definition.yml [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+            minInstances: 0
+            maxInstances: 2
+            enabled: true
+      #List of IP CIDRs to whitelist for public load balancers AND k8s management API
+      ip_addresses: 
+        - "108.5.26.129/32"
+################################################################################
+#Data Warehouse Configurations
+################################################################################
+dw: #No current configs can be set here (yet)
```
Comment (Chaffelson) [inline examples/sandbox-aws/definition.yml:175]: include df

**Review by Chaffelson — CHANGES_REQUESTED** [review]

**Review by wmudge — APPROVED** [review]

**Pull request merged**